interface/ceed-basis.c

```diff
@@ -391,6 +391,7 @@ int CeedMatrixMultiply(Ceed ceed, const CeedScalar *mat_A,
   @param num_comp    Number of field components (1 for scalar fields)
   @param P_1d        Number of nodes in one dimension
   @param Q_1d        Number of quadrature points in one dimension
+  @param basis_space 1 for H^1 discretization (2 for H(div), 3 for H(curl))
```

Comment:
```suggestion
```
Not actually a parameter. (`make doxygen` warns, though you have to troll through the logs to find it).